js/views/scrollView.js

```diff
@@ -772,7 +773,15 @@ ionic.views.Scroll = ionic.views.View.inherit({
         }
       }
 
-      self.doTouchMove(getEventTouches(e), e.timeStamp, e.scale);
+      if(self.options.locking){
+        if(!self.__scrollLockFix.direction){
+          self.__scrollLockFix.direction = Math.abs(e.touches[0].pageX - self.__initialTouchLeft) > Math.abs(e.touches[0].pageY - self.__initialTouchTop);
```

Comment:
Setting direction to `false` to indicate Y direction doesn't seem right. It will still fail the `!self.__scrollLockFix.direction` test in subsequent calls.